utilities/constants.py

```diff
@@ -10,8 +10,12 @@ class ModelFormat:
     CAIKIT: str = "caikit"
 
 
+class ModelName:
+    FLAN_T5_SMALL: str = f"flan-t5-small-{ModelFormat.CAIKIT}"
```

Comment:
Maybe create a constant for `flan-t5-small` and reuse

Reply:
done